scripts/native.proj

```diff
@@ -39,18 +40,30 @@
 
       <_ProjectArgs32> /p:Configuration=$(Configuration) /p:Platform=Win32</_ProjectArgs32>
       <_ProjectArgs64> /p:Configuration=$(Configuration) /p:Platform=x64</_ProjectArgs64>
+      <_ProjectArgsARM64> /p:Configuration=$(Configuration) /p:Platform=ARM64</_ProjectArgsARM64>
     </PropertyGroup>
 
     <!-- Build GitExtSshAskPass project, x86 -->
     <Exec
+        Condition="!$(Arm64Build)"
```

Comment:
This is error-prone as we don't control the value. We should do this instead:
```suggestion
        Condition=" '$(Arm64Build)' != 'true' "
```

Ditto for all other similar instances.

Reply:
I disagree. `Arm64Build` is meant to be a `boolean` flag, with `true` value being the only value having significant effect; any other value is simply `not true` i.e. `false`. I'm just following MSBuild docs here. The fact that `Arm64Build` is implemented as a string is just a shortcoming of MSBuild, so making it explicit like `'$(Arm64Build)' != 'true'` is clunky and unnecessary, in my opinion, as I want to hide its silly string implementation!

Reply:
`dotnet publish /p:Arm64Build=yes` will cause this to fail.
MSBuild doesn't have strong typing, and because of that we should be defensive in our implementations. This is how comparisons are done throughout the .NET SDK.

Reply:
> `dotnet publish /p:Arm64Build=yes` will cause this to fail.

Yes, and it should fail for Arm64 build. As I mentioned before, `Arm64Build` is a boolean flag with `true` having significant meaning, any other values are `false`.

Reply:
MSBuild properties aren't strongly typed. So they have to be compared in a safe way.